benchmarks/res/update.txt

```diff
@@ -0,0 +1,151 @@
+BenchmarkUpdate/114_(5_14_5_5_10_goleveldb)-8          	  200000	     10123 ns/op	    1760 B/op	      26 allocs/op
+BenchmarkUpdate/115_(12_14_5_5_10_goleveldb)-8         	  100000	     16537 ns/op	    4335 B/op	      55 allocs/op
+BenchmarkUpdate/116_(19_14_5_5_10_goleveldb)-8         	   50000	     29607 ns/op	    8157 B/op	      88 allocs/op
+BenchmarkUpdate/117_(5_0_10_5_10_goleveldb)-8          	   30000	     44853 ns/op	    7314 B/op	      90 allocs/op
+BenchmarkUpdate/118_(12_0_10_5_10_goleveldb)-8         	   10000	    126993 ns/op	   22512 B/op	     309 allocs/op
+BenchmarkUpdate/119_(19_0_10_5_10_goleveldb)-8         	   10000	    233959 ns/op	   37109 B/op	     512 allocs/op
+BenchmarkUpdate/120_(5_7_10_5_10_goleveldb)-8          	  200000	     10121 ns/op	    1760 B/op	      26 allocs/op
+BenchmarkUpdate/121_(12_7_10_5_10_goleveldb)-8         	  100000	     16664 ns/op	    4335 B/op	      55 allocs/op
+BenchmarkUpdate/122_(19_7_10_5_10_goleveldb)-8         	   50000	     29509 ns/op	    8157 B/op	      88 allocs/op
+BenchmarkUpdate/123_(5_14_10_5_10_goleveldb)-8         	  200000	     10513 ns/op	    1760 B/op	      26 allocs/op
+BenchmarkUpdate/124_(12_14_10_5_10_goleveldb)-8        	  100000	     17120 ns/op	    4335 B/op	      55 allocs/op
+BenchmarkUpdate/125_(19_14_10_5_10_goleveldb)-8        	   50000	     29944 ns/op	    8158 B/op	      88 allocs/op
+BenchmarkUpdate/126_(5_0_5_10_10_goleveldb)-8          	   20000	     90703 ns/op	   17718 B/op	      90 allocs/op
+BenchmarkUpdate/127_(12_0_5_10_10_goleveldb)-8         	   10000	    208419 ns/op	   51995 B/op	     335 allocs/op
+BenchmarkUpdate/128_(19_0_5_10_10_goleveldb)-8         	   10000	    323295 ns/op	   74366 B/op	     485 allocs/op
+BenchmarkUpdate/129_(5_7_5_10_10_goleveldb)-8          	  200000	     10657 ns/op	    1760 B/op	      26 allocs/op
+BenchmarkUpdate/130_(12_7_5_10_10_goleveldb)-8         	  100000	     21965 ns/op	    5564 B/op	      60 allocs/op
+BenchmarkUpdate/131_(19_7_5_10_10_goleveldb)-8         	   50000	     30621 ns/op	    7754 B/op	      84 allocs/op
+BenchmarkUpdate/132_(5_14_5_10_10_goleveldb)-8         	  200000	     11085 ns/op	    1760 B/op	      26 allocs/op
+BenchmarkUpdate/133_(12_14_5_10_10_goleveldb)-8        	  100000	     22514 ns/op	    5564 B/op	      60 allocs/op
+BenchmarkUpdate/134_(19_14_5_10_10_goleveldb)-8        	   50000	     36978 ns/op	    7753 B/op	      84 allocs/op
+BenchmarkUpdate/135_(5_0_10_10_10_goleveldb)-8         	   20000	    100634 ns/op	   17718 B/op	      90 allocs/op
+BenchmarkUpdate/136_(12_0_10_10_10_goleveldb)-8        	   10000	    224892 ns/op	   51994 B/op	     335 allocs/op
+BenchmarkUpdate/137_(19_0_10_10_10_goleveldb)-8        	   10000	    347126 ns/op	   74366 B/op	     485 allocs/op
+BenchmarkUpdate/138_(5_7_10_10_10_goleveldb)-8         	  100000	     11531 ns/op	    1760 B/op	      26 allocs/op
```

Comment:
Can you add what the numbers are in the benchmark name. Having to refer back to the benchmark code to understand this is inefficient.